### PR TITLE
Add blocks for customizing the TOC drawer

### PR DESCRIPTION
**Pull request by adborden**
- Typo with the reg-head height, it should contain the sub-head height
- Adds block for the sub-head drawer icons
- Adds block for sub-head content

### CODE REVIEW

File: regulations/templates/regulations/chrome.html [hunk 1]
```diff
@@ -33,6 +33,7 @@
 {% include "regulations/main-header.html" %}
 
 <div class="sub-head {% block sub-head-class %}{% endblock %}">
+    {% block sub-head-drawer-icons %}
     <a href="#content-wrapper" class="hidden-text">Skip to main content</a> <!-- skip to content link for screen readers-->
```
Comment (cmc333333) [inline regulations/templates/regulations/chrome.html]: Should this block begin _after_ the skip-to-main? It's an accessibility helper

**Comment by cmc333333** [comment]
Small question, but otherwise works for me

File: regulations/templates/regulations/chrome.html (hunk 1 above)
Comment (adborden) [inline regulations/templates/regulations/chrome.html]: Hmmm, in our case, we're emptying the block and moving the drawer icons somewhere else. In our case it makes sense to move the skip nav along with the drawer icons. I think it's okay to leave the skip-nav within the block, but maybe a) we should pick a better block name or b) put the skip-nav in it's own block?

IMHO I think it's okay to leave as is.

Comment (cmc333333) [inline regulations/templates/regulations/chrome.html]: I like the idea of putting the skip-nav in it's own block; for most use cases, it should hang around even when the drawer icons aren't present.

Comment (adborden) [inline regulations/templates/regulations/chrome.html]: Quoting CM Lubinski (2016-06-14 15:09:34)
> I like the idea of putting the skip-nav in it's own block; for most use cases, it should hang around even when the drawer icons aren't present.

Fixed :)


--
Aaron D Borden
Human and Hacker

**Comment by cmc333333** [comment]
:+1:

**Pull request merged**